src/main/java/net/authorize/util/XmlUtility.java

```diff
@@ -38,15 +42,23 @@ private XmlUtility() {
 	 * @throws IOException if errors during serialization
 	 * @throws JAXBException if errors during serialization
 	 */
-	public static <T extends Serializable> String getXml(T entity) throws IOException, JAXBException
+	public static synchronized <T extends Serializable> String getXml(T entity) throws IOException, JAXBException
 	{

```

Comment:
will not this cause blocking now since we have added synchronized? however, what object are we synchronizing on? IU do not see any lock on object.